examples/column_name_labeler.ipynb

```diff
@@ -0,0 +1,329 @@
+    "2. Run the labeler\n",
+    "\n",
+    "First, let's import the libraries needed for this example."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a67c197b-d3ee-4896-a96f-cc3d043601d3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import sys\n",
+    "import json\n",
+    "from pprint import pprint\n",
+    "\n",
+    "import pandas as pd\n",
+    "\n",
+    "try:\n",
+    "    import dataprofiler as dp\n",
+    "except ImportError:\n",
+    "    sys.path.insert(0, '../..')\n",
+    "    import dataprofiler as dp"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c71356f4-9020-4862-a1e1-816effbb5443",
+   "metadata": {},
+   "source": [
+    "## Loading and using the pre-existing column name labeler\n",
+    "We can easily import the exsting `ColumnName` labeler via the `load_with_components` command from the `dp.DataLabeler`. This allows us to import labeler other than the default structured / unstructured labelers which exist in the library."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "818c5b88",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "parameters = {\n",
+    "            \"true_positive_dict\": [\n",
+    "                {\"attribute\": \"ssn\", \"label\": \"ssn\"},\n",
+    "                {\"attribute\": \"suffix\", \"label\": \"name\"},\n",
+    "                {\"attribute\": \"my_home_address\", \"label\": \"address\"},\n",
+    "            ],\n",
+    "            \"false_positive_dict\": [\n",
+    "                {\n",
+    "                    \"attribute\": \"contract_number\",\n",
+    "                    \"label\": \"ssn\",\n",
+    "                },\n",
+    "                {\n",
+    "                    \"attribute\": \"role\",\n",
+    "                    \"label\": \"name\",\n",
+    "                },\n",
+    "                {\n",
+    "                    \"attribute\": \"send_address\",\n",
+    "                    \"label\": \"address\",\n",
+    "                },\n",
+    "            ],\n",
+    "            \"negative_threshold_config\": 50,\n",
+    "            \"positive_threshold_config\": 85,\n",
+    "            \"include_label\": True,\n",
+    "        }\n",
+    "\n",
+    "label_mapping = {\"ssn\": 1, \"name\": 2, \"address\": 3}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9098329e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# pre processor \n",
+    "preprocessor = dp.labelers.data_processing.DirectPassPreprocessor()\n",
+    "\n",
+    "# model\n",
+    "model = dp.labelers.column_name_model.ColumnNameModel(\n",
+    "    parameters=parameters,\n",
+    "    label_mapping=label_mapping,\n",
+    ")\n",
+    "\n",
+    "\n",
+    "# post processor\n",
+    "postprocessor = dp.labelers.data_processing.ColumnNameModelPostprocessor()"
```

Comment:
Instead of manually loading, could we just use a `load_from_library`? or at least show that at the top so users know they can use a built-in labeler?